src/admin-views/seating/components/layout-card.php

```diff
@@ -4,5 +4,38 @@
  *
  * @since TBD
  *
- * @var Layout_Card $card The Seat Layout card to render.
+ * @var Object $card The Seat Layout card to render.
```

Comment:
Do not make this a generic object: keep it a dedicated value-object class to make sure the API, getter methods, are easy to parse.